platform/values.yaml

```diff
@@ -416,6 +418,9 @@ tagging-pdp:
     oidcSecretName: platform-tagging-pdp-secret
     ## @param tagging-pdp.config.oidcClientId OIDC Client ID for tagging pdp
     oidcClientId: shp-tagging-pdp
+  ## @skip tagging-pdp.securityContext
+  securityContext:
+    runAsUser: 0
```

Comment:
theres an issue w/ taggingpdp grpc gwy container using a non root user, it doesnt have access to some of the swagger files it needs, making bug ticket for this